diff.py

```diff
@@ -1769,7 +1770,7 @@ class ArchSettings:
     name="ppc",
     re_int=re.compile(r"[0-9]+"),
     re_comment=re.compile(r"(<.*>|//.*$)"),
-    re_reg=re.compile(r"\$?\b([rf][0-9]+)\b"),
+    re_reg=re.compile(r"(\$?\b([rf][0-9]+)\b|\(.+\))"),
```

Comment:
Parentheses should not be included in register names. Use lookbehinds/lookaheads if you need to. Also this should only need to capture `0` which should make the regex easier to understand.

Do we actually need to change this regex though? If we don't it will treat `0` vs real register as an instruction diff, which is imo okay because they are categorically different. That's similar to how `$sp` isn't in re_regs for MIPS.

Reply:
I don't think we need this regex changed anymore. I will change it back